packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,432 @@
+
+Two additional `_links` will be returned, one for the schema and another for the User Type. See example below.
+
+```json
+{
+  "_links": {
+    "activate": {
+      "href": "https://{yourOktaDomain}/api/v1/users/00u6d60iOU2PvPYya0g4/lifecycle/activate",
+      "method": "POST"
+    },
+    "schema": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/schemas/user/oscolnyE70oiSYul90g3"
+    },
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/users/00u6d60iOU2PvPYya0g4"
+    },
+    "type": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/types/user/otyolnyE70oiSYul90g3"
+    }
+  },
+  "activated": null,
+  "created": "2018-06-11T18:56:20.000Z",
+  "credentials": {
+    "...": "..."
+  },
+  "id": "00u6d60iOU2PvPYya0g4",
+  "lastLogin": null,
+  "lastUpdated": "2018-08-20T23:42:06.000Z",
+  "passwordChanged": null,
+  "profile": {
+    "...": "..."
+  },
+  "status": "STAGED",
+  "statusChanged": null,
+  "type": {
+    "id": "otyolnyE70oiSYul90g3"
+  }
+}
+```
+
+## User Type Model
+
+### Example User Type
+
+```json
+{
+  "id": "otyfnly5cQjJT9PnR0g4",
+  "displayName": "Freelancer",
+  "name": "Freelance Contractors",
+  "description": "The road goes ever on and on",
+  "createdBy": "00ubade0d00f1f73b234",
+  "lastUpdatedBy": "00ubade0d00f1f73b234",
+  "default": false,
+  "created": "2013-07-02T21:36:25.344Z",
+  "lastUpdated": "2013-07-02T21:36:25.344Z",
+  "_links": {
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4"
+    },
+    "schema": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/schemas/user/oscfnly5cQjJT9PnR0g4"
+    }
+  }
+}
+```
+
+### User Type Properties
+
+The User Type model defines several properties:
+
+| Property      | Description                                       | DataType                                                       | Nullable | Unique | Readonly |
+| ------------- | ------------------------------------------------- | -------------------------------------------------------------- | -------- | ------ | -------- |
+| id            | Unique key for the User Type                      | String                                                         | FALSE    | TRUE   | TRUE     |
+| displayName   | The display name for the type                     | String                                                         | FALSE    | FALSE  | FALSE    |
+| name          | The name for the type                             | String                                                         | FALSE    | TRUE   | FALSE    |
+| description   | A human-readable description of the type          | String                                                         | FALSE    | FALSE  | FALSE    |
+| createdBy     | The user id of the creator of this type           | String                                                         | FALSE    | FALSE  | TRUE     |
```

Comment:
Are we sure we want to include `createdBy` and `lastUpdatedBy`? Looking at our Schemas APIs we don't have those included and this is the last chance we have to remove it so want to make sure this is something we want.

Reply:
Good catch.  Indeed, we do not appear to document these fields for other Models, even though the fields do appear in the responses when the objects are fetched.  This sounds like an Okta-wide API documentation question and I don't know who should answer it. I'll tag @brianzuzga-okta here to see if he wants to offer an opinion.

Reply:
I think this is more a matter of us not having updated the reference since those properties were added.

Reply:
Filed OKTA-249910 to add these to all the other Models.